db/structure.sql

```diff
@@ -461,8 +461,8 @@ CREATE TABLE public.estimated_populations (
     region_id uuid NOT NULL,
```

Comment:
Unrelated changes here.

Reply:
When I pull the master and ran migration these changes were present. I suppose someone forgot to commit these changes. I had discussed with @prabhanshuguptagit whether to commit these changes here. He asked me to commit them.

Reply:
Can we make this another PR, in that case? It's strange that these are getting missed, and we should deal with it separately.

Reply:
Okay. I will raise a separate PR for this